lib/factory.rb

```diff
@@ -1 +1,94 @@
 # frozen_string_literal: true
+class Factory
+  def self.new(*attributes, &block)
+    class_name_from_string = attributes.shift.capitalize if attributes.first.is_a?(String)
+    class_name = Class.new do
+      attributes.each do |attribute|
```

Comment:
`attr_accessor *attributes`

Reply:
On this line, Rubocop gives the following warning: "Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the * if it should be a multiplication."